src/CurrencyInput/index.jsx

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2017-2019 dunnhumby Germany GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file
+ * in the root directory of this source tree.
+ *
+ */
+
+import React                                     from 'react';
+import PropTypes                                 from 'prop-types';
+
+import { attachEvents, mapAria, generateId }     from '../utils';
```

Comment:
`attachEvents` is not implemented, while `mapAria` can be removed (there are no styles)

Reply:
> * `value` prop is not working 🤔(state is updating on input, but if you set `value`, nothing happens)

Thank you for catching this, I'm on it.